tests/foreman/cli/test_computeresource.py

```diff
@@ -291,3 +291,53 @@ def test_update_negative(self, options):
         options.pop('new-name', None)
         for key in options.keys():
             self.assertEqual(comp_res[key], result.stdout[key])
+
+    @data('true', 'false')
+    def test_set_console_password_v1(self, set_console_password):
+        """@Test: Create a compute resource with ``--set-console-password``.
+
+        @Feature: Compute Resource
+
+        @Assert: The option is accepted, and the passed-in value can be read
```

Comment:
I guess you don't want to read it back here, but only with the second test? If so the docstring needs to be updated.